.github/workflows/test-mixed-versions.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
     runs-on: ubuntu-20.04
     env:
       # otp-min
-      otp_version_id: 25_3
+      otp_version_id: 26
```

Comment:
This should probably stay `25_3` as it's the compile-time version of the secondary umbrella, not the runtime.

Reply:
Furthermore, since we don't name these archives after their erlang version, if the same secondary umbrella version is shared across release branches (as is temporarily the case now) it may break the other workflow.